DOCUMENTATION.md

```diff
@@ -1,6 +1,6 @@
 
 
-<!-- Start src/jsonapi-datastore.js -->
+<!-- Start src\jsonapi-datastore.js -->
```

Comment:
Guess my machine automatically changed that when I ran gulp. I made sure it got reverted.